tests/qasm3/test_if.py

```diff
@@ -198,7 +267,7 @@ def test_incorrect_if():
            h q;
            measure q->c;
 
-           if(c[0] >= 1){
+           if(c[0] >> 1){
```

Comment:
Is this a typo?

Reply:
I supported >= now? Maybe I'm misreading this. Shouldn't this pass?

Reply:
Ah gotcha, my bad. Didn't see that this was testing an incorrect if.